components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -325,7 +325,11 @@
             "Key Manager is not enabled in the system"),
     KEY_MANAGER_MISSING_REQUIRED_PROPERTIES_IN_APPLICATION(901407, "Required application properties are missing", 400,
             "Required application properties are missing"),
-    KEY_MAPPING_ALREADY_EXIST(901408, "Application already Registered", 409, "Application already Registered");
+    KEY_MAPPING_ALREADY_EXIST(901408, "Application already Registered", 409, "Application already Registered"),
+
+    //Analytics related codes
+    ANALYTICS_NOT_ENABLED(901500, "Analytics Not Enabled", 404,
+            "Analytics should be enabled to view Bot Detected Data");
```

Comment:
"Bot detection data is not accessible"

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/8708/commits/a053882910460fe518c13a17458690efc528ee4f